### DISCUSSION TITLE
Duplicate query button in the query field should have a limit

### DISCUSSION THREAD

**Discussion by SanthoshS20** [grafana/grafana#53711] [Observability]
**Problem/Idea:**
In the Loki query dashboard, duplicate query button was present in each of the query field. This duplicate query button doesn't have a limit. Anyone can create many duplicates of a single query field. 

**Impact:**
This impacts the performance of the dashboard, took long time to provide the results of all the query 

**Solution:**
The duplicate query button should have a limit. Maximum of 3 query fields can be duplicated. After its reached the maximum limit, the duplicate button should be disabled and should not process the duplicate query API (This API testing needs to be executed in the POSTMAN tool after the code change is done).

**Image:**
![image](https://user-images.githubusercontent.com/37366893/184528126-39fdcc6a-25fb-494d-a464-3ffa93552264.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
